shared/actions/settings.tsx

```diff
@@ -726,6 +729,10 @@ function* settingsSaga(): Saga.SagaGenerator<any, any> {
     SettingsGen.deleteAccountForever,
     deleteAccountForever
   )
+  yield* Saga.chainAction<ConfigGen.BootstrapStatusLoadedPayload>(
```

Comment:
I think it's easier to follow causation if `loadSettings` listens for this action itself

Reply:
hmm, I prefer this way because `loadSettings` would then have to check logged-in-ness in only some cases.

Reply:
i dont think this is a safe assumption

Reply:
what is the assumption?

Reply:
That this gets called only when your logged in

Reply:
currently the function that is called in this `chainAction` is checking for logged-in-ness before returning the loadSettings action.

Reply:
Doesn't mean you won't get logged out. Do your closure relies on being logged in it's safer to validate it yourself vs relying on something upstream